nearcore/src/state_sync.rs

```diff
@@ -87,19 +92,38 @@ pub fn spawn_state_sync_dump(
             )
             .unwrap();
             let arbiter_handle = actix_rt::Arbiter::new().handle();
-            assert!(arbiter_handle.spawn(state_sync_dump(
-                shard_id as ShardId,
-                chain,
-                epoch_manager.clone(),
-                shard_tracker.clone(),
-                runtime.clone(),
-                chain_id.clone(),
-                dump_config.restart_dump_for_shards.clone().unwrap_or_default(),
-                external.clone(),
-                dump_config.iteration_delay.unwrap_or(Duration::from_secs(10)),
-                account_id.clone(),
-                keep_running.clone(),
-            )));
+            match dump_config.multi_node {
```

Comment:
The `multi_node` mode gets the epoch dumped even if there is only one node.
I suggest to remove the alternative non-cooperative codepath and make `multi_node` code path a non-configurable default option.